spec/controllers/spree/checkout_controller_spec.rb

```diff
@@ -29,6 +30,13 @@
           get :edit, params: { state: 'address' }
           expect(response).to redirect_to spree.checkout_registration_path
         end
+
+        context 'and locale is not default' do
```

Comment:
no need to say "and" at start of a context description, it's assumed that a nested context will be "in addition to"